pkg/minikube/cluster/cluster.go

```diff
@@ -104,10 +105,10 @@ func StartHost(api libmachine.API, config MachineConfig) (*host.Host, error) {
 func StopHost(api libmachine.API) error {
 	host, err := api.Load(constants.MachineName)
 	if err != nil {
-		return err
+		return errors.New(err.Error())
```

Comment:
I reckon so, yes, as it means adding context to an error that otherwise would just be a stack trace? Not 100% sure, but feels right.